app/vmalert/main.go

```diff
@@ -47,6 +47,7 @@ eg. 'explore?orgId=1&left=[\"now-1h\",\"now\",\"VictoriaMetrics\",{\"expr\": \"{
 
 	remoteReadLookBack = flag.Duration("remoteRead.lookback", time.Hour, "Lookback defines how far to look into past for alerts timeseries."+
 		" For example, if lookback=1h then range from now() to now()-1h will be scanned.")
+	remoteReadIgnoreRestoreErrors = flag.Bool("remoteRead.ignoreRestoreErrors", true, "Whether to ignore errors from remote storage when restoring alerts state on startup.")
```

Comment:
With default value of `true`, its not possible to use short syntax - `-remoteRead.ignoreRestoreErrors`.
 
 Probably, its better to use default value of `false` with something like:
`remoteRead.failOnRestoreErrors`?

Reply:
Hm, I don't like the proposed flag name. And can't think about any other appropriate name :-) Let's keep it as is.

Reply:
Ok, np)